frontend/app/abtests/BundleVariant.scala

```diff
@@ -9,18 +9,17 @@ object BundleVariant {
   val cookieName = "ab-bundle"
 
   val all = Seq[BundleVariant](
-    BundleVariant(A, 1, Map((Supporter, 6.5), (DigitalPack, 12), (Saturday, 18), (GuardianWeekly, 18), (Weekend, 25), (SatGW, 25), (SixDay, 45), (SevenDay, 50))),
-    BundleVariant(B, 1, Map((Supporter, 7.5), (DigitalPack, 14), (Saturday, 18), (GuardianWeekly, 18), (Weekend, 25), (SatGW, 25), (SixDay, 45), (SevenDay, 50))),
-    BundleVariant(B, 2, Map((Supporter, 7.5), (DigitalPack, 14), (Saturday, 18), (GuardianWeekly, 18), (Weekend, 25), (SatGW, 25), (SixDay, 45), (SevenDay, 50)), hasAdFree = false)
+    BundleVariant(CONTROL, Map((Supporter, 6.5), (DigitalPack, 12), (Saturday, 18), (GuardianWeekly, 18), (Weekend, 25), (SatGW, 25), (SixDay, 45), (SevenDay, 50))),
+    BundleVariant(VARIANT, Map((Supporter, 6.5), (DigitalPack, 12), (Saturday, 18), (GuardianWeekly, 18), (Weekend, 25), (SatGW, 25), (SixDay, 45), (SevenDay, 50)), hasAdFree = false)
   )
 
   def lookup(name: String): Option[BundleVariant] = {
     all.find(_.testId == name)
   }
 }
 
-case class BundleVariant(distribution: Distribution, priceIndex: Int, prices: Map[BundleTier, Double], hasAdFree: Boolean = true) {
-  val testId =  s"MEMBERSHIP_AB_THRASHER_UK_${distribution.name}$priceIndex"
+case class BundleVariant(distribution: Distribution, prices: Map[BundleTier, Double], hasAdFree: Boolean = true) {
+  val testId =  s"MEMBERSHIP_A_THRASHER_UK_${distribution.name}"
```

Comment:
We might want to include commenting as part of this test identifier - eg "MEMBERSHIP_A_COMMENT_THRASHER_UK_..." - in case we do any other kinds of tests with the 'A' variant. Otherwise it gets a bit more fiddly to distinguish test results from different tests.